README.md

```diff
@@ -0,0 +1,7 @@
+# Genetic algorithms
+
+In order to run type in shell 
+```
+pip install -e .
+genetic_alborithms
+```

```

Comment:
W sumie to będzie najważniejsza zmiana. 
Po zmergowaniu tego brancha można będzie uruhcamiać packzę w taki sposób